### PR TITLE
Updates to common code and utility providers.

### PR DESCRIPTION
**Pull request by a-ilango**
- util: Use both fabric name and provider to search fabric
- prov/rxm and rxd: Don't set domain, prov, fabric names.
- util: Rename logging macros for check info functions.

### CODE REVIEW

File: prov/util/src/util_fabric.c [hunk 1]
```diff
@@ -61,7 +61,7 @@ int ofi_fabric_init(const struct fi_provider *prov,
 	atomic_initialize(&fabric->ref, 0);
 	dlist_init(&fabric->domain_list);
 	fastlock_init(&fabric->lock);
-	fabric->name = prov_attr->name;
+	fabric->name = user_attr->name;
```
Comment (shefty) [inline prov/util/src/util_fabric.c]: I need to examine the context around this call, but is this a safe assignment to make?  We're setting a pointer to a pointer provided by the app?

Comment (a-ilango) [inline prov/util/src/util_fabric.c]: Thanks for catching this. User could free user_attr->name. I'll add an strdup() here.

Comment (a-ilango) [inline prov/util/src/util_fabric.c]: Fixed.

File: prov/util/src/util_fabric.c [hunk 2]
```diff
@@ -61,7 +62,7 @@ int ofi_fabric_init(const struct fi_provider *prov,
 	atomic_initialize(&fabric->ref, 0);
 	dlist_init(&fabric->domain_list);
 	fastlock_init(&fabric->lock);
-	fabric->name = prov_attr->name;
+	fabric->name = strdup(user_attr->name);
```
Comment (shefty) [inline prov/util/src/util_fabric.c]: We're going to need an error check too.

**Pull request merged**